default_prompts.py

```diff
@@ -29,7 +29,7 @@
     CHECK_CHEAT = NO_CHECK_CHEAT
 
 problem_multistep_python = (
-    f"""### Spec: In {LANG}, write a function  that returns the sum of the unique prime factors of 
+    f"""### Spec: In {LANG}, write a function sum_unique_prime_factors(n) that returns the sum of the unique prime factors of 
 a given natural number. Example: if the input number was 28, the function would return 9 (2 + 7), 
```

Comment:
do you want to mention n as the given natural number>

Reply:
good idea!